src/features/nft/portfolio/PortfolioPage.tsx

```diff
@@ -19,49 +20,47 @@ import Modal6 from 'features/nft/nftModals/CopiesDetailsModal'
 import Modal7 from 'features/nft/nftModals/PricePlanModal'
 import Modal8 from 'features/nft/nftModals/OwnershipHistoryModal'
 
-export type TPortfolioPage = {
-  currencyName: string
-}
-
-const nft: TNFT = {
-  id: 230456346,
-  title: 'Super nfty floating head professional',
-  image: nftImage,
-  pastelRareness: 65,
-  internetRareness: 45,
-  likes: 721,
-  liked: true,
-  views: 231,
-  type: 'Live Auction',
-  status: 8,
-  price: 12000,
-  currencyName: 'PSL',
-  time: dayjs().add(1, 'day').add(11, 'hours').add(11, 'seconds'),
-  bids: 12900,
-  author: {
-    avatar: avatar1,
-    name: 'Banksy86',
-  },
-  copies: 1,
-  royalty: '10% Perpetual Royalty',
-  owner: 'Banksy86',
-  collection: 'Angel in the sky',
-  category: 'Illustration',
-  tags: ['Add your tag'],
-  description:
-    'There is something magical about space and astronauts. Who wouldn’t want to see the earth from space, or run around on the moon? Space is the final frontier for mankind and essentialy to the future of our species. This piece is an hommage to the brave men and women who represent mankind in this most ambitious of endeavors, who strive to push',
-}
-
 const breadcrumbs = [
   { title: 'My NFT Portfolio', link: '#' },
   { title: '“Super Nfty Floating Head”' },
 ]
 
-nft.description += nft.description // make it longer for "read more"
+export default function PortfolioPage(): JSX.Element {
```

Comment:
Can we use Arrow function for this component?

Reply:
@sanjanadhikari1 please take a look at this comment https://github.com/pastelnetwork/pastel-electron-wallet/pull/186#discussion_r674171369